lib/tabs/utils.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @param {String} tabInterfaceId A tab interface ID
+ * @param {String} tabId A tab ID
+ * @returns {String} A tab element ID. It's a compound of a tab ID
+ *                   and ID of a tabbed interface it belongs to
+ *                   (to prevent bugs when more tabbed interfaces
+ *                    with some accidentally identical tab IDs are
+ *                    used on the same page)
+ */
+export function getTabElementId(tabInterfaceId, tabId) {
+  return tabInterfaceId + '-' + tabId;
+}
+
+/**
+ * @param {String} tabInterfaceId A tab interface ID
+ * @param {String} tabId A tab ID
+ * @returns {String} A corresponding tab panel element ID
+ */
+export function getTabPanelElementId(tabInterfaceId, tabId) {
+  return getTabElementId(tabInterfaceId, tabId) + '-panel';
+}
+
+/**
+ * @param {HTMLElement} parent A parent element
+ * @returns {HTMLElement, null} The first focusable child of the parent
+ *                              element or `null` when none is found
+ */
+export function getFirstFocusableChild(parent) {
```

Comment:
Out of scope of this PR, but I wonder if there might be future use cases for making these general utilities, within KDS or usable in Kolibri, to help with consistent focus management

Reply:
That'd be nice. For now, at least I think I could move this particular utility higher in the structure since it's not coupled to tabs logic like the previous two functions.

Reply:
Done